README.md

```diff
@@ -20,7 +25,7 @@ github:
 
 accounts:
   - address: archway1l2al7y78500h5akvgt8exwnkpmf2zmk8ky9ht3
-    chainId: constantine-3
+    chainName: archwaytestnet
```

Comment:
I can/t find `archwaytestnet3` in chain-registry. Are you refering tohttps://github.com/cosmos/chain-registry/blob/master/testnets/archwaytestnet/chain.json#L3

Reply:
I mean start following chain-registry naming scheme e.g. https://github.com/cosmos/chain-registry/tree/master/testnets/celestiatestnet3, since we plan of syncing to chain-registry soon. archwaytestnet is also fine to be in the docs hre but will give the impression of referring to constantine-1

Reply:
I thought we are following https://github.com/cosmos/chain-registry/blob/master/testnets/osmosistestnet/chain.json#L3 which points to `osmo-test-5` and for older osmoistestnet there is `osmosistesnet4` https://github.com/cosmos/chain-registry/blob/master/testnets/osmosistestnet/chain.json#L3

Reply:
recommendation from chain registry maintainer:

> Networktestnet (unnumbered) is ideal for the current/primary test network, and then incrementing the suffix as the chain id changes